tests/foreman/api/test_errata.py

```diff
@@ -1406,22 +1407,68 @@ def test_errata_installation_with_swidtags(
     """
     module_name = 'kangaroo'
     version = '20180704111719'
-    # setup base_rhel8 and sat_tools repos
-    rhel8_contenthost.create_custom_repos(
-        **{
-            'baseos': settings.repos.rhel8_os.baseos,
-            'appstream': settings.repos.rhel8_os.appstream,
-        }
+    # new cv version for ak
+    cv_publish_promote(
+        target_sat,
+        module_sca_manifest_org,
+        module_cv,
+        module_lce,
     )
-    function_repos_collection_with_manifest.setup_virtual_machine(rhel8_contenthost)
-
-    # install older module stream
+    _ak = target_sat.api.ActivationKey(
+        organization=module_sca_manifest_org,
+        environment=module_lce,
+        content_view=module_cv,
+    ).create()
+    # needed repos for module stream, swid tags, prereqs
+    _repos = {
+        'base_os': settings.repos.rhel8_os.baseos,
+        'sat_tools': settings.repos.rhel8_os.appstream,
+        'swid_tags': settings.repos.swid_tag.url,
+    }
+    # associate repos with host, org, lce, and sync
+    rhel8_contenthost.create_custom_repos(**_repos)
```

Comment:
I'm a bit confused here, do we need the `swid_tags` repo as a custom repo on the host? Because below we add the same repos at the Sat side, publish/promote and consume on the host through AK. I might be missing something.

Reply:
without `chost.create_custom_repos`, setup prereq command fails to install swid-tools ([Ln 1365](https://github.com/SatelliteQE/robottelo/blob/dea50d9a4d50db6ce1e4fbdebeb4b74d1d97fe69/tests/foreman/api/test_errata.py#L1356)), 
or I will get failure to sync repos in `setup_org_for_a_custom_repo`.

Reply:
Doesn't it mean the repo synced to Satellite is not actually used in this test, rather the custom repo created on this line is being used?

Reply:
@lhellebr yup good catch! I have resolved this locally and removed unneeded method `create_custom_repos`